nfcttrace/README.md

```diff
@@ -0,0 +1,19 @@
+# nfcttrace
+
+This traces TCP and UDP entries in nf_conntrack.
+This is done by instrumenting kprobe:nf_ct_delete.
+
+## Output
```

Comment:
Mind adding more detail about what the output is and how it can be used to do X?